README.md

```diff
@@ -25,10 +25,10 @@ gulp.task('default', function () {
 });
 ```
 
-The contains function accepts either a string or an array of strings (any of
+The contains function accepts either a string, regex or an array of strings (any of
 which, when matched, will cause an error to be thrown).
 
-You can also specify a callback function, in which you can handle the error
+You can also specify an onFound or onNotFound callbacks, in which you can handle the error
```

Comment:
This doesn't make sense, it should either by "onFound or onNotFound callbacks" or "an onFound or onNotFound callback" (preferably the former)